platform/ios/src/MGLMapView.mm

```diff
@@ -4091,6 +4091,10 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             {
                 if ([annotation isKindOfClass:[MGLMultiPoint class]])
                 {
+                    if ( ! ((MGLShape *)annotation).enabled) {
```

Comment:
`MGLPointAnnotation` is a form of `MGLShape` annotations if we set the `enable` property to `NO` on a point annotation I would expect that is going to be disabled. This line is never reached in that case.

Reply:
Thanks for pointing that out!